setup.py

```diff
@@ -5,16 +5,16 @@
 # We don't declare our dependency on transformers here because we build with
 # different packages for different variants
 
-VERSION = "0.5.0"
+VERSION = "0.5.1"
 
 # Ubuntu packages
 # libsndfile1-dev: torchaudio requires the development version of the libsndfile package which can be installed via a system package manager. On Ubuntu it can be installed as follows: apt install libsndfile1-dev
 # ffmpeg: ffmpeg is required for audio processing. On Ubuntu it can be installed as follows: apt install ffmpeg
 # libavcodec-extra : libavcodec-extra  includes additional codecs for ffmpeg
 
 install_requires = [
-    "transformers[sklearn,sentencepiece,audio,vision,sentencepiece]==4.44.0",
-    "huggingface_hub[hf_transfer]==0.24.5",
+    "transformers[sklearn,sentencepiece,audio,vision,sentencepiece]==4.46.0",
+    "huggingface_hub[hf_transfer]<0.26",
```

Comment:
du we need this since `diffusers>0.29.0`?

Reply:
The container will be used on new deployments. I think its fine if they break as it seems the decision was to rather help them upgrade their diffusers version etc.

Reply:
Great then reverting that and updating the PR description before merging, thanks!